scripts/mutect2_wdl/unsupported/mutect2_opt.wdl

```diff
@@ -352,56 +423,61 @@ task MergeVCFs {
 }
 
 task CollectSequencingArtifactMetrics {
-  Int preemptible_attempts
-  String m2_docker
   File tumor_bam
   File tumor_bam_index
   File ref_fasta
   File ref_fasta_index
   File picard_jar
 
+  # Runtime parameters
+  String m2_docker
+  Int? preemptible_attempts
+  Int disk_space_gb
+  Int? mem
+
+  # Mem is in units of GB but our command and memory runtime values are in MB
+  Int machine_mem = if defined(mem) then mem * 1000 else 7000
+  Int command_mem = machine_mem - 500
+
   command {
-        set -e
-        java -Xmx4G -jar ${picard_jar} CollectSequencingArtifactMetrics I=${tumor_bam} O="metrics" R=${ref_fasta} VALIDATION_STRINGENCY=LENIENT
+    # Sed'ing of the metrics file to make it GATK compatitble has been moved to the FilterByOrientationBias task
 
-        # Convert to GATK format
-        sed -r "s/picard\.analysis\.artifacts\.SequencingArtifactMetrics\\\$PreAdapterDetailMetrics/org\.broadinstitute\.hellbender\.tools\.picard\.analysis\.artifacts\.SequencingArtifactMetrics\$PreAdapterDetailMetrics/g" \
-            "metrics.pre_adapter_detail_metrics" > "gatk.pre_adapter_detail_metrics"
+    java -Xmx${command_mem}m -jar ${picard_jar} CollectSequencingArtifactMetrics I=${tumor_bam} O="gatk" R=${ref_fasta} VALIDATION_STRINGENCY=LENIENT
```

Comment:
The up to date picard jar isn't in the docker? Should we be using a standard genomes in the cloud docker for this task instead?

Reply:
They might have had a reason that they want to have a input jar here maybe?  @LeeTL1220 any ideas if this was a purposeful decision?

Reply:
I just want to be sure people use the updated picard jar with the faster SequencingArtifactMetrics :)